config.js

```diff
@@ -394,9 +394,17 @@ config.MIN_MEMORY_FOR_UPGRADE = 1200 * 1024 * 1024;
 //////////////////////////////
 
 config.SEND_EVENTS_REMOTESYS = true;
-config.PROMETHEUS_ENABLED = true;
+config.CONFIG_JS_PROM_ENABLED = true;
```

Comment:
You don't need to set CONFIG_JS_ prefix, it will be stripped from env, right?

Reply:
done